src/App.tsx

```diff
@@ -1,9 +1,23 @@
-import React from 'react';
+import React, { useState, ChangeEvent } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+
+  const handleQueryChange = (e: ChangeEvent<HTMLInputElement>) => {
+    setQuery(e.target.value);
+  };
+
+  const visibleMovies = moviesFromServer.filter(movie => {
+    const title = movie.title.toUpperCase();
+    const description = movie.description.toUpperCase();
```

Comment:
You can destructure movie 
```suggestion
  const visibleMovies = moviesFromServer.filter(({ title, description }) => {
    const title = title.toUpperCase();
    const description = description.toUpperCase();
```

Reply:
done